src/pages/public_registration/PublicRegistrationFile.tsx

```diff
@@ -65,32 +67,51 @@ const PublicRegistrationFile: FC<PublicRegistrationFileProps> = ({ file }) => {
     setIsLoadingFile(false);
   };
 
+  const selectedLicense = licenses.find((license) => license.identifier === file.license?.identifier);
+
   return (
     <StyledSidebarCard>
-      <StyledFileIcon />
-      <NormalText>{file.name}</NormalText>
-      {file.embargoDate && new Date(file.embargoDate) > new Date() ? (
-        <StyledNormalText>
-          <LockIcon />
-          {t('will_be_available')} {new Date(file.embargoDate).toLocaleDateString()}
-        </StyledNormalText>
-      ) : !currentFileUrl ? (
-        <StyledButtonWithProgress
-          isLoading={isLoadingFile}
-          endIcon={!isLoadingFile && <CloudDownloadIcon />}
-          data-testid="button-download-file"
-          onClick={() => handleDownload(file.identifier)}>
-          {t('download')}
-        </StyledButtonWithProgress>
-      ) : (
-        <StyledButton
-          variant="contained"
-          color="primary"
-          endIcon={<OpenInNewIcon />}
-          data-testid="button-open-file"
-          onClick={() => window.open(currentFileUrl)}>
-          {t('open')}
-        </StyledButton>
+      <div>
+        <StyledFileIcon />
+        <NormalText>{file.name}</NormalText>
+        {file.embargoDate && new Date(file.embargoDate) > new Date() ? (
+          <StyledNormalText>
+            <LockIcon />
+            {t('will_be_available')} {new Date(file.embargoDate).toLocaleDateString()}
+          </StyledNormalText>
+        ) : !currentFileUrl ? (
+          <StyledButtonWithProgress
+            isLoading={isLoadingFile}
+            endIcon={!isLoadingFile && <CloudDownloadIcon />}
+            data-testid="button-download-file"
+            onClick={() => handleDownload(file.identifier)}>
+            {t('download')}
+          </StyledButtonWithProgress>
+        ) : (
+          <StyledButton
+            variant="contained"
+            color="primary"
+            endIcon={<OpenInNewIcon />}
+            data-testid="button-open-file"
+            onClick={() => window.open(currentFileUrl)}>
+            {t('open')}
+          </StyledButton>
+        )}
+      </div>
+      {selectedLicense && (
+        <div>
+          <img src={selectedLicense.image} alt={selectedLicense.identifier} />
+          <Typography>
+            {selectedLicense.link ? (
+              <Link href={selectedLicense.link} target="_blank" rel="noopener noreferrer">
+                {selectedLicense.label}
+              </Link>
+            ) : (
+              selectedLicense.label
+            )}
+          </Typography>
+          <StyledNormalText>{selectedLicense.description}</StyledNormalText>
+        </div>
```

Comment:
<img width="384" alt="Skjermbilde 2021-02-03 kl  13 08 33" src="https://user-images.githubusercontent.com/6595982/106745113-f3311980-6620-11eb-9053-aae31c190e10.png">

Fikser du mobilvisning i samme slengen? eller evt skriver det opp som egen task?

Reply:
jepp, tar det boks for boks etterpå. Blir en PR per TODO
![bilde](https://user-images.githubusercontent.com/6313468/106745922-14ded080-6622-11eb-932a-0207d5456b13.png)